tmt-app/src/Components/CareerGuide.js

```diff
@@ -1,12 +1,191 @@
-import React from "react";
-import { Container } from "@mui/system";
-import Breadcrumb from "./Breadcrumb";
+import React, { useState, lazy, Suspense } from "react";
+import {
+  Autocomplete,
+  Box,
+  Container,
+  Grid,
+  InputAdornment,
+  TextField,
+  ToggleButton,
+  ToggleButtonGroup,
+  Typography,
+} from "@mui/material";
+import { FlexContainer } from "../Styling/CustomStyling.js";
+import SearchIcon from "@mui/icons-material/Search";
+import MilkTeaSwiggle2 from "../Images/MilkTeaSwiggle2.png";
+import ThaiTeaSwiggle2 from "../Images/art/Thai_Tea_Swiggle_2.png";
+const MajorCard = lazy(() => import("./MajorCard.js"));
+const MajorCareerList = lazy(() => import("./MajorCareerList.js"));
+const GuideSidebar = lazy(() => import("./GuideSidebar.js"));
```

Comment:
We may also not necessarily need this granular code splitting. The size of these assets i think are pretty negligible and probably adding just a little extra overhead than we need right now. I think code splitting at the router level is a safe place to start